### PR TITLE
Move data folder outside content

### PR DESCRIPTION
**Pull request by joaoburatto**
## Summary

As title says.

### CODE REVIEW

**Comment by stilnat** [comment]
What's the reason for this ?

**Review by stilnat — CHANGES_REQUESTED** [review]
wait for #995 before merge.

**Comment by joaoburatto** [comment]
> What's the reason for this ?

New organization definition.

**Pull request closed without merging**
